fetch-utils.js

```diff
@@ -27,3 +27,34 @@ export async function signOutUser() {
 }
 
 /* Data functions */
+
+
+export async function createItem(item) {
+    return await client.from('shopping_list').insert(item);
+
+}
+
+export function renderItem(item) {
+    const li = document.createElement('li');
+    li.textContent = item.quantity + ' ' + item.item;
+    return li;
+}
+
+export async function getListItems() {
+    return await client.from('shopping_list').select();
+
+}
+
+export async function gotIt(someId) {
+    const updatedItem = await client.from('shopping_list').update({ bought: true }).match({ id: someId });
+    return updatedItem;
+}
+
+export async function removeItems() {
+    const user = getUser();
+    return await client.from('shopping_list').delete().eq('user_id', user.id);
+}
```

Comment:
it'd be better to write this as
```
export async function removeItems(id) {
    return await client.from('shopping_list').delete().eq('user_id', id);
}
```
then you only have to call ```getUser()``` once in ```app.js``` and store it in a state variable ```user``` then when you call it you could call ```removeItems(user.id)```